Makefile

```diff
@@ -21,17 +24,27 @@ with_azure_deps := $(shell grep -i 'install_azure_dependencies' $(conf_dir)/conf
 .DEFAULT_GOAL := work
 
 check:
-ifneq ($(shell test -z ${TERRAFORM_WRAPPER_SHELL}; echo $$?),0)
```

Comment:
This should be kept.

Reply:
👌 but test shall be changed to something valid in fish like for line 27